2.11.0-M3

retronym opened the pull request:
Review by @adriaanm

retronym committed:
A little housekeeping
 - add a .gitignore
 - pin down the SBT version.

retronym committed:
Release notes for 2.11.0-M3

 - Correct Scala IDE section, after input from @dragos.
 - Improved the bug reporting section
 - Use the range v2.11.0-M1-v2.11.0-M3

Generated the commit list from the merge base of master and 2.10.x
and the soon-to-be-tagged 01dec25425@scala/scala.

retronym committed:
Mandate UTF-8 as file.encoding.

A better fix would avoid use of the platform default encoding
altogether.

Also adds a meta directive the the generated HTML to set the
encoding to UTF-8. We assume that the page that this is embedded
into follows the same conventions, either with a meta tag or with
a HTTP header.

eed3si9n commented on hand-written.md:
```diff
@@ -1,85 +1,50 @@
+    * Infer bounds of existential types [SI-1786](https://issues.scala-lang.org/browse/SI-1786)
+* REPL
+    * The bytecode decompiler command, :javap, now works with Java 7 [SI-4936](https://issues.scala-lang.org/browse/SI-4936) and has sprouted new options [SI-6894](https://issues.scala-lang.org/browse/SI-6894) (Thanks, Andrew Marki!)
+    * Added command :kind to help to tell ground types from type constructors. [#2340](https://github.com/scala/scala/pull/2340) (Thanks, George Leontiev and Eugene Yokata!)
```
Typo on my last name: Yokota.

adriaanm replied:
I'd also consider turning people's names into links to their github profile.

retronym replied:
What, and risk more `[]` `()` permutations!?

Just kidding, that's a good idea. Thanks for the fine toothcomb review!

adriaanm commented on hand-written.md:
```diff
@@ -1,85 +1,50 @@
+      were shipped. We have now removed the old implementations.
+    * scala-actors is now deprecated; we advise users to follow the steps in the [Actors Migration Guide](http://docs.scala-lang.org/overviews/core/actors-migration-guide.html) to port to Akka Actors, which have been included in
+      the distribution since 2.10.0.
+    * Search and destroy mission for ~5000 chunks of dead code. (#1648)[https://github.com/scala/scala/pull/1648/files]
```
Swapped braces: [#1648](https://github.com/scala/scala/pull/1648/files)

adriaanm commented on src/main/scala/MakeReleaseNotes.scala:
```diff
@@ -41,6 +41,7 @@ object MakeReleaseNotes {
 
     s"""<html>
       <head>
+        <meta charset="UTF-8">
```
:heart:  thanks for fixing the weird letters! :neckbeard:

retronym committed:
Incorporate review feedback

  - Scala IDE is releasing for Indigo.
  - Link to external contributors' profiles
  - Markdownoes

retronym commented:
@adriaanm Updated

/cc @JamesIry

retronym commented:
@adriaanm I don't have permissions to merge this.

retronym committed:
Make mention of two known bugs in this milestone.

adriaanm commented:
> @adriaanm I don't have permissions to merge this.

@retronym that should be fixed now
